packages/terra-clinical-item-collection/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 * Changed
+  * Updated jest snapshots terra-icon A11y updatess.
```

Comment:
We can remove this as it is no longer needed with the dangerfile changes.

Reply:
removed [here](https://github.com/cerner/terra-clinical/pull/823/commits/7cb1cc54d09c6b0f308fdeb4e38f55dd39d88f4c#diff-2836fb928f9d253961572dc46bc45f787fc9c2abb9f2b56679f04387c4d80c2e)